osc-server/src/main/java/org/osc/core/broker/service/validator/DeleteDistributedApplianceRequestValidator.java

```diff
@@ -53,6 +60,10 @@ public DistributedAppliance validateAndLoad(BaseDeleteRequest request) throws Ex
                             da.getId()));
         }
 
+        // DE5296->Error message to be more appropriate when deleting a DA without
```

Comment:
remove comments related to the issue - DES296.

Reply:
Thank you for your comments . I removed the comments and will be updated in next review request.